lib/errors.ts

```diff
@@ -140,10 +140,14 @@ const messages: {
 	BalenaExpiredToken: () => stripIndent`
 		Looks like the session token has expired.
 		Try logging in again with the "balena login" command.`,
+
+	BalenaAmbiguousDevice: () => stripIndent`
+		Error: Multiple devices found whose UUID starts with the provided UUID.`,
```

Comment:
Re _"the provided UUID"_ - as a global error handler in `errors.ts`, we don't know where exactly this error was raised. The error may have come from a dependency in `node_modules`. Also, some CLI commands allow multiple device UUIDs to be typed:
```
$ balena env add EDITOR vim --device 7cf02a6,d6f1433
```
... and the user would not know which of the many UUIDs had a problem if we print "the provided UUID". The default error message for `BalenaAmbiguousDevice` actually includes the offending UUID:

> https://github.com/balena-io-modules/balena-errors/blob/v4.7.1/lib/errors.ts#L336
> `Device is ambiguous: ${device}`

So I'm thinking we might be better off not translating the error message. This would mean deleting the lines above, but keeping the entry in `EXPECTED_ERROR_REGEXES`. I think the user would then get an error message similar to:

```
$ balena device deactivate a782d09
BalenaAmbiguousDevice: Device is ambiguous: a782d09

$ balena env add EDITOR vim --device 7cf02a6,a782d09
BalenaAmbiguousDevice: Device is ambiguous: a782d09
```
It not the nicest error message, but at least it points to the offending UUID.